lib/chef/knife/ec_backup.rb

```diff
@@ -226,7 +226,7 @@ def download_org_data(name)
           top_level_acls  = chef_fs_paths('/acls/*.json', chef_fs_config, [])
           acl_paths       = chef_fs_paths('/acls/*/*', chef_fs_config, exclude_list)
           group_paths     = chef_fs_paths('/groups/*', chef_fs_config, exclude_list)
```

Comment:
are these being used anywhere after the change?

Reply:
The other thing to look at is if this prevents us from excluding billing-admins and public_key_read_access.

Reply:
@marcparadise Can you elaborate on excluding billing-admins and public_key_read_access?  Can you point me to something explaining how to do it? (checked docs on `knife ec backup/restore` but didn't see anything).

Reply:
does not look like the top_level_acls, acl_paths and group_paths are being used anywhere. They can be deleted if they are not.